### PR TITLE
Website Brand Redesign

### PR DESCRIPTION
**Pull request by Prid13**
#### Style

- Primary Buttons colors changed
- All black boxes changed to aurora green, with internation blue text on top
  - Also, some tweaks have been made to accomodate for this change (some transparency values, color, borders, etc.)
- Light blue pills changed to light aurora green

##### Images
- New Logo added
- All instances of Logos changed (social media image, favicons, etc.)
- Author image added for wgtwo

##### Fonts
- English and Japanese fonts separated (Inter for en, Noto Sans JP for jp)
  - NB: wasn't able to limit the JP font to only JP characters, so the whole JP website uses a different font. To solve this, a custom font would have to be self-hosted on WG2 servers to limit the JP font to only JP characters, and thus Google's servers wouldn't be used (not sure how good of a solution is this)

#### Names
- "Working Group Two" => "WG2"
  - Q: was I supposed to do this? or only wgtwo?
- "wgtwo" => "WG2"

###### Exceptions
- Some Quotes
  - E.g., look at the huge quote on the front page by CHK, it says "Our collaboration with Working Group Two"
    -  Q: should this also be changed? They didn't outright say it, so I'm wondering how "nazi" we should be 😅
  - In other quotes I've changed the wording to WG2 (bc I was lazy and didn't wanna proofread lol)
- Legal instances ("Working Group Two AS")
- old blog posts mentioning "Working Group Two"/"Wgtwo"
  - Q: should they be untouched for posterity/integrity?

#### URLs
- wgtwo.com => wg2.com
- github.com/working-group-two/wgtwo.com => github.com/working-group-two/**wg2**.com

###### Exceptions
- subdomains like developer.wgtwo.com, trust.wgtwo.com, gk.wgtwo.com, ip.wgtwo.com, etc.
  - NB: there are quite a few of these scattered everywhere, in code and in blog posts. They're probably used in external services by third parties, so should they be changed as well?
- external services like wgtwo.jobs.personio.de

---

#### Things to Consider

- Github account names: e.g. jonny-wgtwo, mtl-wgtwo
- Netlify name: wgtwo-dev.netlify.app
- "wgtwo" blog author name (will have to manually change for each relevant blog post)
- Emails: e.g. erlend@wgtwo.com
- Social media handles: e.g. @workinggrouptwo (Twitter)
- publickey.txt (WG2's public key for authentication. It has "Working Group Two AS" branded in it)

---

#### Screenshots

Screenshot_6428

Screenshot_6429

Screenshot_6430

Screenshot_6431

---

### Alt button border color:

**Look at the blue border on the buttons** -- what do you think?

Screenshot_6434
Screenshot_6435

---

### What to do with these?

Screenshot_6433

Screenshot_6436

### CODE REVIEW

**Comment by Prid13** [comment]
Build Website is failing bc I've changed "wgtwo.com" to "wg2.com" and also the github repo name to "wg2.com". Ignore it

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -2,14 +2,14 @@ const path = require("path")
...
 module.exports = {
-  title: "Working Group Two",
+  title: "WG2",
   tagline:
-    "No more boxes, no more end of life, no large upfront investments. With the wgtwo cloud native mobile core we offer continuous innovation, continuous development, scalability and flexibility. And to make it really simple for operators – delivered as a managed SaaS telecommunication network.",
+    "No more boxes, no more end of life, no large upfront investments. With the WG2 cloud native mobile core we offer continuous innovation, continuous development, scalability and flexibility. And to make it really simple for operators – delivered as a managed SaaS telecommunication network.",
   url: "https://www.wgtwo.com",
   baseUrl: "/",
   favicon: "img/favicons/favicon.ico",
-  organizationName: "working-group-two",
```
Comment (jonny-wg2) [inline website/docusaurus.config.js:11]: keep this for now

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -2,14 +2,14 @@ const path = require("path")
...
-  projectName: "wgtwo.com",
```
Comment (jonny-wg2) [inline website/docusaurus.config.js:12]: keep this wgtwo.com for now. we will do a staged deployment.

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -233,7 +233,7 @@ module.exports = {
           path: "../docs",
           sidebarPath: require.resolve("./sidebars.js"),
           editUrl:
-            "https://github.com/working-group-two/wgtwo.com/edit/main/docs/",
```
Comment (jonny-wg2) [inline website/docusaurus.config.js:236]: keep for now

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -244,7 +244,7 @@ module.exports = {
           postsPerPage: 18,
           blogSidebarCount: 1000,
           editUrl:
-            "https://github.com/working-group-two/wgtwo.com/edit/main/blog/",
```
Comment (jonny-wg2) [inline website/docusaurus.config.js:247]: keep for now

**Review by jonny-wg2 — CHANGES_REQUESTED** [review]
Great work @Prid13  - it's a lot!!

I left a couple comments. Let's make the changes to revert some lines in the docusaurus.config.js. It would be best to do a staged deployment where we make this go live with wgtwo.com and then we make the changes to change the domain to wg2.com

**Comment by Prid13** [comment]
@jonny-wg2 thanks, done :)

**Pull request closed without merging**